tests/unit/plugins/modules/system/test_parted.py

```diff
@@ -298,7 +298,7 @@ def test_create_label_gpt(self):
             '_ansible_check_mode': True,
         })
         with patch('ansible_collections.community.general.plugins.modules.system.parted.get_device_info', return_value=parted_dict2):
-            self.execute_module(changed=True, script='unit KiB mklabel gpt mkpart primary 0% 100% unit KiB name 1 \'"lvmpartition"\' set 1 lvm on')
+            self.execute_module(changed=True, script='unit KiB mklabel gpt mkpart primary 0% 100% unit KiB name 1 lvmpartition set 1 lvm on')
```

Comment:
This looks wrong, it should probably be
```suggestion
            self.execute_module(changed=True, script='unit KiB mklabel gpt mkpart primary 0% 100% unit KiB name 1 "lvmpartition" set 1 lvm on')
```